aws/resource_aws_route53_vpc_association_authorization.go

```diff
@@ -0,0 +1,135 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/route53"
+)
+
+func resourceAwsRoute53VPCAssociationAuthorization() *schema.Resource {
+	return &schema.Resource{
```

Comment:
This resource should implement import support. Please see the [Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/contributing/contribution-checklists.md#adding-resource-import-support) for more information about how to accomplish this with testing and documentation.

In this case, using the `schema.ImportStatePassthrough` helper should work along with the below changes.

```go
Importer: &schema.ResourceImporter{
	State: schema.ImportStatePassthrough,
},
```